### PR TITLE
Lambda Layer Update - Latest Published Layer ARNs for Jan 2022 release

### PR DESCRIPTION
**Pull request by vasireddy99**
Description:

Updates the layer.tf and template.yml file to have the latest Lambda Layer ARNs for the layers released as part of Jan 2022 release.

This includes OTel Python 1.9.1 & ADOT Collector 0.43.1

Link to tracking Issue:

N/A

Testing:

Covered by Soaking Tests.

Documentation:

N/A

### CODE REVIEW

**Review by wangzlei — APPROVED** [review]

File: python/sample-apps/aws-sdk/deploy/wrapper/layer_amd64.tf [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
+  "eu-west-2"      = "arn:aws:lambda:eu-west-2:901920570463:layer:aws-otel-python38-amd64-ver-1-9-1:1"
+  "eu-west-3"      = "arn:aws:lambda:eu-west-3:901920570463:layer:aws-otel-python38-amd64-ver-1-9-1:1"
+  "sa-east-1"      = "arn:aws:lambda:sa-east-1:901920570463:layer:aws-otel-python38-amd64-ver-1-9-1:1"
+  "us-east-1"      = "arn:aws:lambda:us-east-1:901920570463:layer:aws-otel-python38-amd64-ver-1-9-1:1"
+  "us-east-2"      = "arn:aws:lambda:us-east-2:901920570463:layer:aws-otel-python38-amd64-ver-1-9-1:1"
+  "us-west-1"      = "arn:aws:lambda:us-west-1:901920570463:layer:aws-otel-python38-amd64-ver-1-9-1:1"
+  "us-west-2"      = "arn:aws:lambda:us-west-2:901920570463:layer:aws-otel-python38-amd64-ver-1-9-1:1"
   }
-}
+}
```
Comment (NathanielRN) [inline python/sample-apps/aws-sdk/deploy/wrapper/layer_amd64.tf:20]: Missing new line

File: sample-apps/python-aws-sdk-aiohttp-sam/template.yml [hunk 2]
```diff
@@ -37,34 +37,34 @@ Resources:
...
     us-east-2:
-      layer: "arn:aws:lambda:us-east-2:901920570463:layer:aws-otel-python38-ver-1-7-1:1"
+      layer: "arn:aws:lambda:us-east-2:901920570463:layer:aws-otel-python38-amd64-ver-1-9-1:1"
     us-west-1:
-      layer: "arn:aws:lambda:us-west-1:901920570463:layer:aws-otel-python38-ver-1-7-1:1"
+      layer: "arn:aws:lambda:us-west-1:901920570463:layer:aws-otel-python38-amd64-ver-1-9-1:1"
     us-west-2:
-      layer: "arn:aws:lambda:us-west-2:901920570463:layer:aws-otel-python38-ver-1-7-1:1"
\ No newline at end of file
+      layer: "arn:aws:lambda:us-west-2:901920570463:layer:aws-otel-python38-amd64-ver-1-9-1:1"
```
Comment (NathanielRN) [inline sample-apps/python-aws-sdk-aiohttp-sam/template.yml:70]: Let's add the new line

File: dotnet/sample-apps/aws-sdk/deploy/wrapper/layer_amd64.tf [hunk 3]
```diff
@@ -1,20 +1,20 @@
 locals {
```
Comment (NathanielRN) [inline dotnet/sample-apps/aws-sdk/deploy/wrapper/layer_amd64.tf:1]: We also need to update `go` layer ARNs!

File: dotnet/sample-apps/aws-sdk/deploy/wrapper/layer_amd64.tf [hunk 4]
```diff
@@ -1,20 +1,20 @@
...
-  collector_layer_arns_amd64 = {
```
Comment (NathanielRN) [inline dotnet/sample-apps/aws-sdk/deploy/wrapper/layer_amd64.tf:2]: We shouldn't change the spacing, let's put these spaces back.

**Review by NathanielRN — CHANGES_REQUESTED** [review]

**Review by NathanielRN — APPROVED** [review]
Thank you for this! Let's go!!

**Pull request merged**